colibri/src/globaldb.rs

```diff
@@ -63,3 +63,58 @@ impl GlobalDB {
         })
     }
 }
+
+/// macro that creates a copy of the globaldb in the rotkehlchen data folder
+/// and stores it in a temp folder
+#[cfg(test)]
+#[macro_export]
+macro_rules! create_globaldb {
```

Comment:
same concerns for the test location here. I guess we would have this in a utils or just globaldb file but under the tests directory